docker/central-ledger/default.json

```diff
@@ -67,6 +67,14 @@
       }
     }
   },
+  "PROXY_CACHE": {
+    "enabled": false,
+    "type": "redis",
```

Comment:
shouldn't we use `redis-cluster` as it's default proxyCache type?

Reply:
let me do that in a follow up PR. I didn't want to touch the int harness.